index.bs

```diff
@@ -1171,7 +1174,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Relying Party Identifier</dfn>
 : <dfn>RP ID</dfn>
-:: A [=valid domain string=] that identifies the [=[WRP]=] on whose behalf a given [=registration=] or
+:: In the context of the [=web authentication api|WebAuthn API=], a [=relying party identifier=] MUST be a [=valid domain string=] identifying the [=[WRP]=] on whose behalf a given [=registration=] or
```

Comment:
The "MUST be" seems a bit off to me... how about "[in the web context] an RP ID _is_ a valid domain string"?

Reply:
I dunno, we've been treating it as a MUST for a long while now, this is just being explicit.

Reply:
Yeah, what I mean is that from an abstract point of view this seems like an odd way to define it. In abstract terms an RP ID just _is_ a domain string (in the context of the Web API, at least). In practical terms the parameter of course uses the larger type "text string" and restricts it to the subset of domain strings, and for that a "MUST be" seems appropriate. But here it feels odd because "RP ID" in the abstract is a new concept rather than a special case of an existing one.

Anyway, this is really really minor nitpicking, feel free to leave this as is if I'm being silly. :smile:

Reply:
I'll `s/ MUST be / is /`.